tests/test_WebAccess.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+from rdflib import Graph
+
+from pytravharv.WebAccess import download_uri_to_store
+
+
+@pytest.mark.usefixtures("target_store_access")
+def test_download_uri_to_graph(target_store_access):
+    uri = "https://www.w3.org/People/Berners-Lee/card.ttl"
+    graph = Graph()
+    download_uri_to_store(uri, graph)
+    assert isinstance(graph, Graph)
+    assert len(graph) > 0
+    target_store_access.ingest(graph, "uri:PYTRAVHARV:base_test.yml")
+    query_results = target_store_access.full_graph()
+    assert len(query_results) > 0
+    assert query_results is not None
+    assert len(query_results) == len(graph)
+
+    # Add more assertions as needed
+
+
+# ttl file
+# jsonld file
+# html file with ttl file in head as full script
+# html file with jsonld file in head as script link
+test_cases = [
```

Comment:
slightly dangerous test-cases as external changes can make these tests fail and suddenly prevent PR-check, package building, ... etc etc

also makes 'dev on the train' rather impracticable  (Belgian trains are still soo off-line)

rather consider launching a localhost http server port (fastapi?) that serves grabbed copies from  those files from ./tests/web/**.*

that would also make sure we control that the strategies of disovery are actually under our control 9so we know what we are effectively testing)

of course -- this is quite a task in itself -- best to push it in a separate issue to some future milestone

Reply:
issue mentioned in #22